meal-planner/src/components/Header.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components'
+import logo from '../assets/logo.svg'
```

Comment:
If you use a SVG and want to see a SVG tag in your HTML, you could also import it as a ReactComponent...

`import {ReactComponent as Logo} from ...`
`<LogoStyled/>`
`const LogoStyled = styled(Logo)..`